### PR TITLE
Hcqis updates

### PR DESCRIPTION
**Pull request by tomwillis608**
Remove HCQIS FC standup list. Update iQIES FC standup list. Add iQIES Scrum of Scrums chooser.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,30 +32,29 @@ function qppfc_standup_list() {
...
-  ];
-  var arr2 = [
-    "tom w", "ryan", "bryon"
+     "chris", "james", "john", "josh", "tim"
   ];
   arr1 = shuffle(arr1); 
-  var arr = arr1.concat(arr2);
-  console.log(arr.join(","));
-  return arr.join(", ");
+  var arr = arr1[0];
```
Comment (tomwillis608) [inline index.js:55]: I don't use JavaScript that often. Would it be more stylish to use arr1.shift() rather than arr1[0]?

Comment (djacobs-flexion) [reply index.js:55]: @matthopson can you make a suggestion?  I am not a js wiz either.

Comment (matthopson) [reply index.js:55]: You can certainly use `shift` to grab that element, however it's destructive to the array, so if you expect the array to be mutated (that element is pulled from it and all the other elements shift their position), then go for it!

Comment (tomwillis608) [reply index.js]: Let's try it with shift()

**Review by djacobs-flexion — APPROVED** [review]

**Pull request merged**